be/src/olap/aggregate_func.h

```diff
@@ -342,6 +418,48 @@ struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_HLL_UNION, OLAP_FIELD_TYPE_HLL
         delete context->hash64_set;
     }
 };
+// when data load, after bitmap_init fucntion, bitmap_union column won't be null
+// so when init, update, finalize bitmap, we needn't consider null
+template <>
+struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_BITMAP_UNION, OLAP_FIELD_TYPE_VARCHAR> {
+    static void init(RowCursorCell* dst, const char* src, bool src_null, Arena* arena) {
+        auto* src_slice = reinterpret_cast<const Slice*>(src);
+        auto* dst_slice = reinterpret_cast<Slice*>(dst->mutable_cell_ptr());
+
+        char* mem = arena->Allocate(sizeof(RoaringBitmap));
```

Comment:
we can just new from heap to make things ease

Reply:
Use arena, in order to we could collect the memory usage.